encoder/vaapiencoder_base.h

```diff
@@ -127,6 +128,11 @@ class VaapiEncoderBase : public IVideoEncoder {
     uint32_t frameRateNum() const {
         return m_videoParamCommon.frameRate.frameRateNum;
     }
+
+    uint32_t fps() const {
+        return m_videoParamCommon.frameRate.frameRateNum / m_videoParamCommon.frameRate.frameRateDenom;
```

Comment:
frameRateDenom has been initialized to be 1 when class VaapiEncoderBase is constructed. 
Nobody will change it to be 0 if he understand what  frameRateDenom means.